### ISSUE TITLE
FlexFlow Release 2020.08

### ISSUE THREAD

**Issue by jiazhihao** [flexflow/FlexFlow#38]
List of TODOs for the next FlexFlow release:
- [x] Runtime support for [Loss](https://keras.io/api/losses/) and [Metrics](https://keras.io/api/metrics/) functions 
- [x] FlexFlow Keras support for Loss and Metrics functions
- [x] Future plans for multiple work branches (stable for future releases, master for internal development)
- [x] unit tests and regression tests for FlexFlow
- [x] (Optional) Finish the integration of the Candle benchmarks
- [ ] (Optional) Docker images of stable builds

TODOs for future releases
- [ ] (Optional) Scalability tests
- [ ] Website
- [ ] Documentations
- [ ] CMake and Spack for stable builds
- [ ] Support for PyTorch lazy model
- [ ] TensorBoard support using [TensorBoardX](https://github.com/lanpa/tensorboardX)
- [ ] Distributed Data Loader in Python, Generator support for Keras
- [ ] Support for AMD CPUs/GPUs
